rotkehlchen/types.py

```diff
@@ -417,6 +417,15 @@ def get_chain_type(self) -> str:
         # else
         return 'eth2'  # the outlier
 
+    def get_name(self) -> str:
+        return SUPPORTED_BLOCKCHAIN_TO_NAME[self]
+
+    def get_native_asset(self) -> str:
```

Comment:
Right now this doesn't return an asset, but just a string. If the asset is needed, I would return `A_ETH`, `A_BTC`, etc. then.

Reply:
I don't mind decoupling native asset from the enum value.
That would probably also imply always returning `native_asset` key in the response.

Reply:
We don't want `native_asset` in every entry. Only when it's different from the short name. So right now only in Optimism.

Where is this function used? If it's only used for this new api, then perhaps does not even make sense to create it. Just add this logic directly in the api endpoint and if we ever need this we would create such a fucntion here.

Cause you would need to indeed return an asset and not a string here. But returning constant assets here will complicate the code and will probably hit you with import problems as this is `types.py` a very central file.

Reply:
updated